test/unit/scheduled_publisher_test.rb

```diff
@@ -32,6 +32,11 @@ class ScheduledPublisherTest < ActiveSupport::TestCase
       assert @edition.reload.published?
     end
 
+    should "should update publishing api with latest edition payload before publishing" do
```

Comment:
The test name states  that it is checking that the publishing API is updated with the latest edition _before_ publishing, but the test doesn't actually check any ordering. If the order is important, it would be nice to check that in the test (or rename the test to not mislead readers).

Reply:
fair point, though ordering is important and thats what we wanted to convey through test, I am not sure if there is a way to test ordering.